Design.md

```diff
@@ -242,6 +243,22 @@ Originally discussed in [#31](https://github.com/coreos/fedora-coreos-tracker/is
  - Ignition spec 3.0.0 will break compatibilty with spec 2.x.y, although most configs will only require minor changes.
  - Tooling should exist to aid converting 2.x.y configs to 3.0.0 configs, although perfect automated translation will not be possible.
 
+## Configuration Language and Transpiler
+
+- Originally discussed in issue [#129](https://github.com/coreos/fedora-coreos-tracker/issues/129).
+- Versioning discussed in issue [#89](https://github.com/coreos/fedora-coreos-tracker/issues/89)
+
+### Summary:
+
+Fedora CoreOS will have a configuration language similar to the [Container Linux Configuration Language]() named the Fedora CoreOS Configuration Language (FCCL). There will be a tool, the Fedora CoreOS Configuration Transpiler (FCCT) to convert Fedora CoreOS Configs (FCCs) to Ignition configs.
```

Comment:
Missing a link to https://coreos.com/os/docs/latest/configuration.html here?

Reply:
yup, just pushed  a fix